src/import/load-costume.js

```diff
@@ -7,7 +7,11 @@ const loadVector_ = function (costume, runtime, rotationCenter, optVersion) {
     if (optVersion && optVersion === 2 && !runtime.v2SvgAdapter) {
         log.error('No V2 SVG adapter present; SVGs may not render correctly.');
     } else if (optVersion && optVersion === 2 && runtime.v2SvgAdapter) {
-        runtime.v2SvgAdapter.loadString(svgString, true /* fromVersion2 */);
+        try {
+            runtime.v2SvgAdapter.loadString(svgString, true /* fromVersion2 */);
```

Comment:
@fsih good point, thanks for catching that. I've moved everything into a try/catch block to make sure we load the default SVG if any of the SVG loading logic fails. Another approach I was thinking of was to use the default SVG at [the point of the original error](https://github.com/LLK/scratch-svg-renderer/blob/develop/src/svg-renderer.js#L84), though then we would need to add scratch-storage as a dependency to scratch-svg-renderer and it wouldn't allow us to update the costume's asset id from scratch-svg-renderer.

Reply:
@fsih good point, thanks for catching that. I've moved everything into a try/catch block to make sure we load the default SVG if any of the SVG loading logic fails. Another approach I was thinking of was to use the default SVG at [the point of the original error](https://github.com/LLK/scratch-svg-renderer/blob/develop/src/svg-renderer.js#L84), though then we would need to add scratch-storage as a dependency to scratch-svg-renderer and it wouldn't allow us to update the costume's asset id from scratch-svg-renderer.

Reply:
I feel like it's not ideal to wrap so much code in a try. Could we limit the try/catch to places where we call v2SvgAdapter.toString?

scratch-svg-renderer shouldn't depend on scratch-storage. It should be limited to a util for performing operations on SVGs.

Reply:
@fsih that makes sense. What do you think of how I have it set up now with wrapping the whole thing in a promise, since we're returning promises (either `reject` or `resolve`) anyways? This way any error from `v2SvgAdapter.toString` get passed up as rejected promises. Alternatively I can add multiple try/catch blocks, this just struck me as a little neater.

Reply:
This looks good!